developers-guide.md

```diff
@@ -30,20 +30,20 @@ Set API base URL in `environments/environment.prod.ts`, you can always change it
 #### Core:
  Library      |   Version     | Latest Stable (? Y/n) | License
 ------------- | ------------- | --------------------  | ------------
- Angular      | 4.4.6         |        5.2.9          | MIT 
- Angular CLI  | 1.6.8         |        1.7.3          | MIT 
- TypeScript   | 2.3.4         |        2.7.2          | Apache 2.0
- rxjs         | 5.4.3         |        5.4.3          | Apache 2.0
+ Angular      | 4.4.6         |        6.0.0          | MIT 
+ Angular CLI  | 1.6.8         |        6.0.1          | MIT 
+ TypeScript   | 2.3.4         |        2.8.3          | Apache 2.0
+ rxjs         | 5.4.3         |        6.1.0          | Apache 2.0
 
 #### Dev:
  Library         |   Version     | Latest Stable (? Y/n) | License
 ---------------- | ------------- | --------------------  | ------------
-chart.js         |  2.7.0        |        2.7.0          | MIT 
-core-js          |  2.5.1        |        2.5.1          | MIT 
-lodash-es        |  4.17.7       |        4.17.4         | MIT
+chart.js         |  2.7.0        |        2.7.2          | MIT 
```

Comment:
should we upgrade to 2.7.2 itself? as its bug fixes release

Reply:
Yes we can do that. I have checked it and it is working fine.